src/features/Producer/utils/producerRouteParams.ts

```diff
@@ -0,0 +1,6 @@
+/** Describes shape of route params in producer feature. */
```

Comment:
Let's keep it inside `Producer/routes.tsx`, so in case if params are changed, the developer would notice and update this interface with higher chance :)

Reply:
It's related to dependency cycle import